### ISSUE TITLE
LaserWriter 4/600 PS prints blank pages when rendered by GhostScript

### ISSUE THREAD

**Issue by mabam** [OpenPrinting/cups-filters#76]
The Apple LaserWriter 4/600 PS seems to suffer from the same bug in its PostScript interpreter as the Apple LaserWriter 12/640 PS does: When GhostScript is used to render the .ps files this results in empty pages printed. For the latter printer this has been fixed by a workaround in cups-filters:
http://bzr.linuxfoundation.org/loggerhead/openprinting/cups-filters/revision/7449 .
Would it be possible to also fix this for the 4/600 PS? For now I've done it manually by adding "Option pdftops-renderer pdftops" to /etc/cups/printers.conf as described at https://bugzilla.redhat.com/show_bug.cgi?id=1304869#c12 and it's working fine.

I wonder whether the Personal LaserWriter 320, Color LaserWriter 12/600 PS, Color LaserWriter 12/660 PS, and LaserWriter 16/600 PS suffer from the same bug. But I can't verify as I don't have access to any of these.

**Comment by tillkamppeter** [comment]
Fixed in commit 91ec37bec. Please test. Thank you for the bug report.

**Comment by mabam** [comment]
Thank you for the quick fixing!

My system is not capable of the newest CUPS version, so it will take a while until I can test.

**Comment by mabam** [comment]
The LW 4/600 PS requires AppleTalk which I use a2server for to provide it. a2server runs on Debian, though not on present versions.
I have now installed the latest Debian/Armbian updates for Jessie on my AppleTalk NAS so it is capable of CUPS 1.7.5. After deleting the old CUPS version I've downloaded the source for cups-filters-1.0.61, patched filter/pdftops.c with your fix from v1.21.6, compiled and installed. Then I installed CUPS 1.7.5 and printed (without "Option pdftops-renderer pdftops" in /etc/cups/printers.conf) …

Working great, thank you again!
